deephaven/vscode-deephaven
# fix: Search filtering now works in PANELS

I fixed a bug where I was setting description instead of label that was breaking built in search / filter in the "PANELS" treeview. Also added search actions to the titlebar of CONNECTIONS + PANELS trees

### Testing
```python
from deephaven import empty_table

for i in range(100):
    print(f"Making {i}")
    globals()[f"t{i}"] = empty_table(10).update(['X = i', 'Y = 10 * i'])
```

- Hover the title of the PANELS tree
- Click the search icon (magnifying glass)
- Type the name of a variable to filter on
- Should scroll to the variable. Clicking the filter button in the search box should actually filter the results

resolves #49
